fix: return environments keyed by environment type (#148)
salesforcecli/plugin-env: messages/list.md

# error.NoResultsFound

No results found.

# warning.RequestedColumnsNotPresentInEnvironment

> The columns specified on the command line (%s) are not present in Environment "%s".

Reviewer: ```suggestion
The columns you specified (%s) aren't available in the table for environment "%s".
```
